### PR TITLE
Use https in dev

### PR DESCRIPTION
**Pull request by peerwit**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -12,7 +12,11 @@ var mongoose = require('mongoose');
 var config = require('./server/config/environment');
 var https = require('https');
 var fs = require('fs');
-var credentials = {key: fs.readFileSync('./shared/config/ssl.key', 'utf-8'), cert: fs.readFileSync('./shared/config/ssl.crt', 'utf-8')};
+console.log(process.env.NODE_ENV);
+var credentials = {
+			key: process.env.NODE_ENV === 'production' ? fs.readFileSync('../shared/config/ssl.key', 'utf-8') : fs.readFileSync('./shared/config/ssl.key', 'utf-8'),
```
Comment (kxprim) [inline server.js]: Indentation is offset

File: server.js [hunk 2]
```diff
@@ -28,22 +32,26 @@ var socketio = require('socket.io')(serverHTTPS, {
   serveClient: (config.env === 'production') ? false : true,
   path: '/socket.io-client'
 });
-app.all('*', function(req,res){
-	console.log('asdf');
+app.all('*', function(req,res, next){
+	if (!req.connection.encrypted) {res.redirect('https://sphereable.com')}
```
Comment (kxprim) [inline server.js]: Fix indentation

File: server.js [hunk 3]
```diff
@@ -28,22 +32,26 @@ var socketio = require('socket.io')(serverHTTPS, {
...
   console.log('Express server listening on %d, in %s mode', config.port, app.get('env'));
 });
 
-serverHTTP.listen(80, config.ip, function () {
-  // console.log(req.connection.encrypted?true:false);
-  console.log('Express server listening on %d, in %s mode', config.port, app.get('env'));
-});
+if (process.env.NODE_ENV === 'production') {
+	serverHTTP.listen(80, config.ip, function () {
+	  // console.log(req.connection.encrypted?true:false);
```
Comment (kxprim) [inline server.js]: Fix indentation, may be check if Sublime settings is set to tabs rather than spaces.

**Comment by scottrice10** [comment]
:+1:

**Comment by kxprim** [comment]
:+1:

**Pull request merged**